### PR TITLE
add sidebar

### PR DESCRIPTION
**Pull request by TMysiv**
add sidebar

### CODE REVIEW

File: src/components/sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    {
+      path: 'about',
+      name: 'About',
+      icon: <InfoIcon fontSize={'large'}/>
+    }
+  ];
+
+  return (
+    <div className={style.container}>
+      <div style={{ width: isOpen ? '250px' : '70px' }} className={style.sidebar}>
```
Comment (lerik1408) [inline src/components/sidebar/Sidebar.tsx:51]: Let's get rid of style={}. We can  add a new class for showing the sidebar
https://bobbyhadz.com/blog/react-add-remove-class-on-click

**Review by lerik1408 — COMMENTED** [review]
GJ

File: src/components/sidebar/Sidebar.tsx (hunk 1 above)
Comment (TMysiv) [reply src/components/sidebar/Sidebar.tsx]: fixed

**Review by lerik1408 — COMMENTED** [review]
GJ

**Pull request merged**